internal/age/ssh.go

```diff
@@ -289,11 +289,8 @@ func ParseSSHIdentity(pemBytes []byte) (Identity, error) {
 		}
 
 		fmt.Print("Enter passphrase for encrypted id_rsa: ")
```

Comment:
This message should use a user-recognizable key identifier, because you can encrypt non-RSA identities and there may be multiple SSH identities on the system. Can you pass the filename down to here?